src/Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk/targets/windows/wix.targets

```
@@ -194,7 +194,7 @@
     the bundle references.
   -->
   <Target Name="GetBundledMsiFiles">
-    <MSBuild Projects="$(RepoRoot)signing\SignMsiFiles.proj" Targets="Build" />
+    <MSBuild Projects="$(SigningToolsDirectory)\SignMsiFiles.proj" Targets="Build" />
```

Redundant `\` (and applying `Dir` change):

```suggestion
    <MSBuild Projects="$(SigningToolsDir)SignMsiFiles.proj" Targets="Build" />
```